packages/trpc/server/routers/viewer/organizations/getFacetedValues.handler.ts

```diff
@@ -0,0 +1,58 @@
+import { RoleManagementFactory } from "@calcom/features/pbac/services/role-management.factory";
```

Comment:
I think this file can be removed. we don't use it anywhere now that we formulate this object directly in RSC (/members/page)
```
 {
    teams: teams,
    attributes: attributes,
    roles: roles,
  }
```

Reply:
Oh yeah nice spot